.github/workflows/ci.yml

```diff
@@ -1,33 +1,32 @@
 name: Crossroad
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   iOS:
-    runs-on: macos
+    runs-on: macos-latest
     steps:
     - uses: actions/checkout@v2
     - name: Run Tests
       run: |
         DESTINATION="platform=iOS Simulator,name=iPhone 8" SCHEME="Crossroad-iOS"
         xcodebuild test -project Crossroad.xcodeproj -scheme "${SCHEME}" -destination "${DESTINATION}"
   tvOS:
-    runs-on: macos
+    runs-on: macos-latest
     steps:
     - uses: actions/checkout@v2
     - name: Run Tests
       run: |
         DESTINATION="platform=tvOS Simulator,name=Apple TV 4K" SCHEME="Crossroad-tvOS"
         xcodebuild test -project Crossroad.xcodeproj -scheme "${SCHEME}" -destination "${DESTINATION}"
   Lint:
-    runs-on: macos
+    runs-on: macos-latest
     steps:
     - uses: actions/checkout@v2
     - name: Install Dependencies
       run: |
         gem install cocoapods
         pod repo update
-        brew install swiftlint
```

Comment:
SwiftLint is preinstalled in the latest virtual macOS environment, and removed the installation step.